pallets/pallet-dip-provider/src/lib.rs

```diff
@@ -31,23 +31,17 @@ pub mod pallet {
 	use frame_support::{pallet_prelude::*, traits::EnsureOrigin};
 	use frame_system::pallet_prelude::*;
 	use parity_scale_codec::FullCodec;
-	use sp_io::MultiRemovalResults;
 	use sp_std::fmt::Debug;
 
-	use crate::traits::{IdentityCommitmentGenerator, IdentityProvider, SubmitterInfo};
+	use crate::traits::{IdentityCommitmentGenerator, IdentityProvider, ProviderHooks, SubmitterInfo};
```

Comment:
You mean that there is a `Config` type which is also called `ProviderHooks`?

Reply:
No, sorry - I'm not referring to the Config. I shouldn't have marked it here.

In this [line](https://github.com/KILTprotocol/kilt-node/pull/574/files#diff-6dc5259ea5e53a98cd0b1ecd6ce9819c2efbb44cb8957070cb0e69b75fbdfa65R65), you're implementing the trait `ProviderHooks` for `FixedDepositCollectorViaDepositsPallet`. When I read the line, my initial assumption was that `ProviderHooks` are from the pallet-deposit-storage. It would be more clear if the trait somehow indicated that the hooks are from the DIP pallet.

Reply:
Mhh what makes you think that `ProviderHooks` is part of the deposit pallet? Because the type implementing it is declared here? Also, the trait is imported from `pallet_dip_provider` with `use pallet_dip_provider::traits::ProviderHooks`. But I anyway renamed the import to avoid confusion, what do you think?  https://github.com/KILTprotocol/kilt-node/pull/574/commits/f73778daaf1dbf38db5d0a6a948698c124001146

Reply:
I believe that 'ProviderHooks' is a broad term that could be associated with many pallets. I think that with the renaming, it becomes more declarative. I needed to scroll up to see where the trait comes from - will be not necessary anymore.